app/javascript/Components/Result/marks_panel.jsx

```diff
@@ -11,47 +11,56 @@ export class MarksPanel extends React.Component {
 
   constructor(props) {
     super(props);
+
+    // Set first criterion active by default
+    const first = !props.released_to_students ? props.marks[0] : null;
```

Comment:
This code should not assume that `props.marks` is non-empty (if it's empty, you can still set `first` to `null`